### ISSUE TITLE
Per subreddit post layouts

### ISSUE THREAD

**Issue by bagarwa** [ccrama/Slide#1133]
Slide version: 4.6.2
Android version: 6.0.1

Right now, "Post Layout" options are present in settings and those layout settings get applied to all subs. Instead of that, it would be better if a "Post Layout" button shows up on the actionbar of each sub's page. Right now the actionbar has refresh, multi-column (I think), sort and menu. If it has "Post Layout" button too, I will be able to switch the layout for different subs on the fly.

I would also hope that the changes get saved. For example, I visit /r/funny and have "big picture" layout. Then I visit /r/eli5 and change the layout to compact. (i.e. disable big pictures). Then I visit /r/funny again. /r/funny should retain its big picture mode while /r/eli5 should retain it's compact mode.

**Comment by ccrama** [comment]
This has been done with the addition of the switch for big pics, and this can apply to any subreddit (including all and frontpage)!
